misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JClass.java

```diff
@@ -91,7 +98,9 @@ private String getExceptionName(Class exception, String name) {
             // Append the prefix "J" in front of bindings generated for Java exceptions.
             if (this.getClass().getClassLoader().loadClass(Exception.class.getCanonicalName())
                     .isAssignableFrom(exception)) {
-                return "J" + name;
+                String shortClassName = "J" + name;
```

Comment:
@IrushiL As "J" letter has been used in several places, can't we use a constant for that?

Reply:
Done